js/utils/getBitcoinPrice.js

```diff
@@ -13,107 +13,115 @@ module.exports = function(currency, callback){
+    var callCoinkite = function ()
+    {
       $.ajax({
         method: "GET",
-        dataType: "json",
-        url: "https://api.coindesk.com/v1/bpi/currentprice/" + currency + ".json"
+        url: "https://api.coinkite.com/public/rates"
       })
           .done(function (response)
           {
-            //console.log("coinDesk: " + response.bpi[currency]['rate']);
-            if($.isNumeric(response.bpi[currency].rate)) {
-              btPrices.push(response.bpi[currency].rate);
+            var coinKiteCurrencies = {};
+            for (var currency in response.rates.BTC) {
+                coinKiteCurrencies[currency] = response.rates.BTC[currency].rate;
             }
+            btPrices.push(coinKiteCurrencies);
           })
           .fail(function (jqXHR, textStatus, errorThrown)
           {
-            //console.log("coinDesk request failed:");
-            //console.log(jqXHR);
-            //console.log(textStatus);
-            //console.log(errorThrown);
+            console.log("coinKite request failed: ");
+            console.log(jqXHR);
+            console.log(textStatus);
+            console.log(errorThrown);
           })
           .always(function ()
           {
-            callBlockchain();
+            callBitcoinAvg();
           });
-    //};
 
+    };
 
-    var callBlockchain = function ()
+    var callBitcoinAvg = function ()
     {
       $.ajax({
         method: "GET",
-        url: "https://blockchain.info/ticker"
+        url: "https://api.bitcoinaverage.com/all"
       })
           .done(function (response)
           {
-            //console.log("blockChain: " + response[currency]['15m']);
-            if($.isNumeric(response[currency]['15m'])) {
-              btPrices.push(response[currency]['15m']);
+            var bitCoinAvgCurrencies = {};
+            for (var currency in response) {
+                if(response[currency].averages) {
+                    bitCoinAvgCurrencies[currency] = response[currency].averages['24h_avg'];
+                }
             }
+            btPrices.push(bitCoinAvgCurrencies);
           })
           .fail(function (jqXHR, textStatus, errorThrown)
           {
-            //console.log("blockChain request failed: ");
-            //console.log(jqXHR);
-            //console.log(textStatus);
-            //console.log(errorThrown);
+            console.log("Bitcoin average request failed: ");
+            console.log(jqXHR);
+            console.log(textStatus);
+            console.log(errorThrown);
           })
           .always(function ()
           {
-            callCoinKite();
+            callBitcoinCharts();
           });
     };
 
-    var callCoinKite = function ()
+    var callBitcoinCharts = function ()
     {
       $.ajax({
         method: "GET",
-        url: "https://api.coinkite.com/public/rates"
+        url: "http://api.bitcoincharts.com/v1/weighted_prices.json"
       })
           .done(function (response)
           {
```

Comment:
bitcoincharts doesn't return JSON, it returns a string, which was causing all the currencies to come back as undefined below. 

For all of the AJAX calls, it's a good idea to check if the response is in JSON, and if not do JSON.parse() on the response before trying to pull data out of it.